GCC warning: enumeral and non-enumeral type in conditional expression [-Wextra]

The code is very suspicious:

```
drivers/gles3/rasterizer_scene_gles3.cpp: In member function 'void RasterizerSceneGLES3::initialize()':
./core/typedefs.h:124:40: warning: enumeral and non-enumeral type in conditional expression [-Wextra]
 #define MIN(m_a, m_b) (((m_a) < (m_b)) ? (m_a) : (m_b))
                        ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
drivers/gles3/rasterizer_scene_gles3.cpp:5042:31: note: in expansion of macro 'MIN'
   state.max_ubo_reflections = MIN(RenderList::MAX_REFLECTIONS, max_ubo_size / sizeof(ReflectionProbeDataUBO));
                               ^~~
```

--- a/drivers/gles3/rasterizer_scene_gles3.cpp
+++ b/drivers/gles3/rasterizer_scene_gles3.cpp
@@ -5053,7 +5053,7 @@ void RasterizerSceneGLES3::initialize() {
 		state.scene_shader.add_custom_define("#define MAX_LIGHT_DATA_STRUCTS " + itos(state.max_ubo_lights) + "\n");
 		state.scene_shader.add_custom_define("#define MAX_FORWARD_LIGHTS " + itos(state.max_forward_lights_per_object) + "\n");
 
-		state.max_ubo_reflections = MIN(RenderList::MAX_REFLECTIONS, max_ubo_size / sizeof(ReflectionProbeDataUBO));
+		state.max_ubo_reflections = MIN((int)RenderList::MAX_REFLECTIONS, max_ubo_size / sizeof(ReflectionProbeDataUBO));
 
 		state.reflection_array_tmp = (uint8_t *)memalloc(sizeof(ReflectionProbeDataUBO) * state.max_ubo_reflections);
 

